knotx-it-tests/src/test/resources/conf/it-tests-application.conf

```diff
@@ -0,0 +1,147 @@
+########### Modules to start ###########
+# You need to specify a list of verticle to be started by Knot.x
+# Each line should have a form of <alias>=<verticle-class-name>
+# where alias is just a name that you can use later in order to define configuration for the module
+# verticle-class-name is a fully qualified class name of the verticle
+#
+# If you want to test the configuration you can simply comment out or remove unnecessary modules
+modules = [
+  ########### Core ###########
+  "server=io.knotx.server.KnotxServerVerticle"
+  "httpRepo=io.knotx.repository.http.HttpRepositoryConnectorVerticle"
+  "fsRepo=io.knotx.repository.fs.FilesystemRepositoryConnectorVerticle"
+  "splitter=io.knotx.splitter.FragmentSplitterVerticle"
+  "assembler=io.knotx.assembler.FragmentAssemblerVerticle"
+  ########### Data Bridge ###########
+  "dataBridge=io.knotx.databridge.core.DataBridgeKnot"
+  "dataSourceHttp=io.knotx.databridge.http.HttpDataSourceAdapter"
+  ########### Forms ###########
+  "forms=io.knotx.forms.core.FormsKnot"
+  ########### Template Engine ###########
```

Comment:
Please add here the comment that Forms adapter is defined and initialized in Test class.